iphone/Maps/Maps.xcodeproj/project.pbxproj

```diff
@@ -4658,7 +4658,7 @@
 				MARKETING_VERSION = 2022.11.17;
 				OTHER_SWIFT_FLAGS = "$(inherited)";
 				PRODUCT_BUNDLE_IDENTIFIER = app.organicmaps.debug;
-				PROVISIONING_PROFILE_SPECIFIER = "CarPlay Debug";
```

Comment:
That is not right. Will debug builds run on CarPlay?

Reply:
I do not understand why this line has been removed. It is required to work with CarPlay in Debug.

Reply:
@biodranik He removed it because of this warning:

<img width="557" alt="Screenshot 2024-05-13 at 16 04 57" src="https://github.com/organicmaps/organicmaps/assets/6050639/98b1cf7d-9ecd-4f52-a9be-1df633e7b80f">

Anyway, I tested CarPlay with auto signing and it worked even without this line.

Reply:
ACK, thanks for the update.

Reply:
What about
```
4683:				PROVISIONING_PROFILE_SPECIFIER = "CarPlay Release";
```
?

Should it also be removed?

Reply:
The release is still on manual signing so I don't think we can remove this line there.